src/tasks/processBulkUpload.ts

```diff
@@ -293,7 +291,7 @@ export const processBulkUpload = async (
 			const proposalVersion = await createProposalVersion({
 				proposalId: proposal.id,
 				applicationFormId: applicationForm.id,
-				sourceId: source.id,
+				sourceId: bulkUpload.sourceId,
```

Comment:
Excellent. What would be nifty, not now, but perhaps someday, is to somehow include "this came from source X via bulk upload" in the provenance, whether that be another source that has an expanded name or is chained to that source or related to that source, etc. Anyway that's way too complex for now, right now this is awesome.